examples/basic/android/settings.gradle

```diff
@@ -1,12 +1,12 @@
 rootProject.name = 'VideoPlayer'
 
-include ':app',
-        ':react-native-video',
-        ':react-native-video-exoplayer'
+include ':react-native-video'
+project(':react-native-video').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-video/android-exoplayer')
```

Comment:
is this correct path? shouldn't it be `'../node_modules/react-native-video/android'`?

Reply:
This will use the new ExoPlayer instead of MediaPlayer, which I think we want.

Reply:
`include ':react-native-video'
project(':react-native-video').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-video/android-exoplayer')`
this is correct

Reply:
i have created new project to test it